reader.go

```diff
@@ -865,8 +864,20 @@ func (r *Reader) run() {
 		l.Printf("entering loop for consumer group, %v\n", r.config.GroupID)
 	})
 
+	attempt := 0
```

Comment:
Actually, can we rename this to `unknownTopicErrCount` since we increment it only in the case of that specific error?